eogrow/core/pipeline.py

```diff
@@ -201,20 +197,20 @@ def run_execution(
         )
         execution_results = executor.run(**executor_run_params)
 
-        successful_eopatches = [eopatch_list[idx] for idx in executor.get_successful_executions()]
-        failed_eopatches = [eopatch_list[idx] for idx in executor.get_failed_executions()]
+        successful = [execution_names[idx] for idx in executor.get_successful_executions()]
```

Comment:
In principle, it was more intuitive before what the `execution_names` object represents. Why not keep the same name? Or is the role of execution names now wider?

Reply:
these names can be whatever the user wants. Before this was given from the EOPatch manager, now they are given from the `get_execution_arguments`. Nothing guarantees patch names.

Reply:
Will the eopatch names be used as a default or will the user have to provide names?

Reply:
Failed to see this was already merged :D